src/test-apps/MockWdmSubscriptionResponder.cpp

```diff
@@ -80,6 +80,8 @@ static bool gCleanStatus = true;
 static nl::Weave::WRMPConfig gWRMPConfig = { kWRMPInitialRetransTimeoutMsec, kWRMPActiveRetransTimeoutMsec, kWRMPAckTimeoutMsec, kWRMPMaxRetrans };
 
 
+#if 0
```

Comment:
Please choose an appropriate preprocessor mnemonic for this. Perhaps `WEAVE_ENABLE_TLV_PRETTY_PRINTER` or some such.

Reply:
TBH, I was somewhat struggling with the justification for keeping this around, this feels like a classic copy-and-paste issue.  I'd prefer to delete it, waiting on any other stakeholders to opine whether such functionality remains useful

Reply:
I proactively removed the block in question.  Rationale: test executables may have a pretty printer available in a different file, as seen in several other files in the test-apps